Makefile

```diff
@@ -105,3 +105,8 @@ endif
 	@echo "api_url: $(TEST_API_URL)" >> $(INTEGRATION_CONFIG)
 	@echo "api_version: $(TEST_API_VERSION)" >> $(INTEGRATION_CONFIG)
 	@echo "ca_file: $(TEST_API_CA)" >> $(INTEGRATION_CONFIG)
+
+inject:
```

Comment:
Just to make sure things stay tidy, it make sense to also add a `inject-clean` that runs `ansible-specdoc -jc -i {FILE}` on each module to clear out the injected docs data.